iOS/Extensions/UserDefaults/UserDefaults+Theme.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import Foundation
+
+enum Theme: Int {
```

Comment:
Instead of basing the enum on `Int`, I would like o see the adoption of the `CaseIteable` protocol.